cypress/e2e/tests/pages/explorer/more-resources/api/custom-resource-definitions.spec.ts

```diff
@@ -126,24 +129,27 @@ describe('CustomResourceDefinitions', { testIsolation: 'off', tags: ['@explorer'
     });
 
     it('sorting changes the order of paginated CRDs data', () => {
+      cy.tableRowsPerPage(10);
```

Comment:
We need to control this for each tests, we don't want a global setting to interfere on other areas, and it's a per session setting.

Reply:
maybe i'm missing something but the setting persists across sessions until updated

Reply:
I think I messed my explanation after revisiting it. The setting **does** persist that's the reason it could cause collisions with other tests if run in parallel. That was the use case I was thinking.